src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,53 @@
+package racingcar.controller;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import racingcar.model.Information;
+import racingcar.model.RacingCar;
+import racingcar.model.RacingCars;
+import racingcar.model.RandomNumberGenerator;
+import racingcar.util.Constants;
+
+public class RacingController {
+    public final RacingCars racingCars;
+    public final int times;
+
+    public RacingController(Information information) {
+        racingCars = init(information.getNames());
+        this.times = information.getTimes();
+    }
+
+    private RacingCars init(List<String> names) {
+        List<RacingCar> racingCars = new ArrayList<>();
+        for (String name : names) {
+            racingCars.add(RacingCar.from(name));
+        }
+        return RacingCars.from(racingCars);
+    }
+
+    public void go() {
+        for (RacingCar car : racingCars.getRacingCars()) {
+            car.move(new RandomNumberGenerator());
```

Comment:
인스턴스 변수에 있는 모델을 가져와서 `getRacingCars()` 를 호출하는 방식이군요!
이러한 방식과 필드값을 선언하지 않고 외부에서 파라미터로 모델을 받는 방식의 차이점은 무엇일까요??
영선님의 생각이 궁금합니다!😁
```suggestion
    public void go(RacingCars racingCars) {
        for (RacingCar car : racingCars.getRacingCars()) {
            car.move(new RandomNumberGenerator());
```

Reply:
이에 관해 공부후 답변드리고 싶어 답변이 늦어졌습니다!
외부에서 파라미터로 모델을 받도록 분리하면, 결국 전체 게임을 관리하는 GameController에서 racingCars를 파라미터로 주입받아야 할 것 같아요.
저는 RacingController로, 레이싱에 관련된 기능들을 분리해 응집도를 높이고자 했습니다.! 

클래스에 인스턴스 변수는 많이 가질수록 안 좋지만, 메소드가 하나 이상의 인스턴스 변수를 사용해 응집도를 높일 수 있었습니다!

Reply:
@oyoungsun  
좋은 의견이네요! 저는 파라미터로 받는 경우 얻을 수 있는 이점이 많다고 생각합니다!

우선 파라미터로 도메인 객체를 받는다면 테스트에서 이점을 얻을 수 있다고 생각해요!
원하는 상태의 도메인 객체을 생성한다면 간단히 `go` 메서드에 파라미터로 넘겨주고 실행해주면 되니까요!
만약 도메인을 인스턴스 변수로 가지고 있는 경우라면 도메인을 원하는 상태로 만들기 위해 `go` 메서드 외의 다른 메서드를 호출해줘야 할 것 같아요! 이는 테스트의 어려움을 주지 않을까 생각이듭니다!

또한 지금 단계에서는 필요없지만 컨트롤러는 실제 웹 프로젝트에서 싱글톤으로 구현됩니다! 즉, 하나의 객체를 여러 사용자가 동시에 사용하게 되는데 만약 인스턴스 변수로 도메인 객체를 가지고 있다면 데이터가 꼬이는 문제가 발생할 수 있을 것 같습니다!